diff --git a/bin/nx.js b/bin/nx.js
index 64d7279..6b55cf2 100644
--- a/bin/nx.js
+++ b/bin/nx.js
@@ -1,6 +1,14 @@
 #!/usr/bin/env node
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
+const events = require("node:events");
+const listenerLimit = 128;
+if (events.defaultMaxListeners < listenerLimit) {
+    events.defaultMaxListeners = listenerLimit;
+}
+if (process.getMaxListeners() < listenerLimit) {
+    process.setMaxListeners(listenerLimit);
+}
 const find_workspace_root_1 = require("../src/utils/find-workspace-root");
 const chalk = require("chalk");
 const dotenv_1 = require("../src/utils/dotenv");
